controllers/scrapeController.js

```diff
@@ -0,0 +1,27 @@
+// controller/scrapeController.js
+const axios = require("axios");
+const cheerio = require("cheerio");
+
+async function scrapeESPN(req, res) {
+  try {
+    console.log("Request received for /scrape-espn");
+    const response = await axios.get("https://www.espn.com");
```

Comment:
move url to a new file called constants.js to store hard-coded variables that may change like urls